### DISCUSSION TITLE
Particle PBD normal

### DISCUSSION THREAD

**Discussion by rossetti73** [NVIDIA-Omniverse/PhysX#165] [Q&A]
hi,
i use PBD cloth but solver seams not calculate normal ?
What is best and fast way to calculate normal about PxPBDParticleSystem?
thanks .

**Comment by preist-nvidia** [comment]
You mean to update the normals in the skin mesh? Yes you will have to implement your own favorite method here. I would assume that you will get a lot of hits if you google this problem.
